main.js

```diff
@@ -68,23 +68,31 @@ app.on('ready', function () {
 
   if (process.platform === 'darwin') {
     // Create our menu entries so that we can use MAC shortcuts
-    Menu.setApplicationMenu(Menu.buildFromTemplate([
-      {
-        label: 'Edit',
-        submenu: [
-          { role: 'undo' },
-          { role: 'redo' },
-          { type: 'separator' },
-          { role: 'cut' },
-          { role: 'copy' },
-          { role: 'paste' },
-          { role: 'pasteandmatchstyle' },
-          { role: 'delete' },
-          { role: 'selectall' }
-        ]
-      }
-    ]));
-  }
+
+    var template = [{
+      label: app.getName(),
+      submenu: [
+        { label: "About " + app.getName(), selector: "orderFrontStandardAboutPanel:" },
+        { type: "separator" },
+        { label: "Quit", accelerator: "Command+Q", click: function () { app.quit(); } }
+      ]
+    }, {
+      label: "Edit",
+      submenu: [
+        { label: "Undo", accelerator: "CmdOrCtrl+Z", selector: "undo:" },
```

Comment:
Why use `CmdorCtrl` here? They should only use `Cmd`. Using `role` as the code did before seems like a much better way to write this.